src/layers/legacy/medRegistration/itkProcessRegistration.cpp

```diff
@@ -174,7 +174,7 @@ template <typename PixelType>
             }
             catch(itk::ExceptionObject &ex)
             {
-                dtkDebug() << "Extraction failed";
+                dtkDebug() << "Extraction failed" << ex.what();
```

Comment:
You should add `:` after `Extraction  failed` to make a clear visual distinction with the appended message.
(same for below)

Reply:
Ok I put : plus 2 spaces